app/src/main/java/org/glucosio/android/tools/InputFilterMinMax.java

```diff
@@ -27,22 +27,22 @@
 public class InputFilterMinMax implements InputFilter {
     private static final String TAG = "InputFilterMinMax";
 
-    private int min, max;
+    private double min, max;
 
-    public InputFilterMinMax(int min, int max) {
+    public InputFilterMinMax(double min, double max) {
         this.min = min;
         this.max = max;
     }
 
     public InputFilterMinMax(String min, String max) {
-        this.min = Integer.parseInt(min);
-        this.max = Integer.parseInt(max);
+        this.min = Double.parseDouble(min);
```

Comment:
This might throw exception

Reply:
Extracted safe parse double utility method